core/src/test/resources/integration_test_files/helloworld/get_api_integration_test.json

```diff
@@ -2,7 +2,7 @@
     "scenarioName": "As simple GET API - Inetgration Test - Local Server",
     "steps": [
         {
-            "name": "find_match",
+            "name": "find_match_1",
```

Comment:
Try reverting to `find_match`

+ Probably this might be failing the build

Reply:
Updated the changes to **find_match**